package.json

```diff
@@ -44,7 +44,8 @@
     "react-scripts": "3.4.1",
     "serverless": "^2.29.0",
     "serverless-finch": "^2.6.0",
-    "typescript": "~3.7.2",
+    "serverless-single-page-app-plugin": "file:./serverless-single-page-app-plugin",
```

Comment:
Suggestion: You don't need to add this plugin to dependencies list. Why did you do this?

Reply:
It was in the learning materials

Reply:
It was in the learning materials 
https://videoportal.epam.com/video/B9Ry76DWWwVng509Y8xq
~3 min 30 sec

Reply:
Okay. But there is more convenient and easiest way. You can put file with plugin inside .serverless_plugins folder and then specify this plugin at plugins section in serverless.yml

https://ibb.co/25KKccc

Reply:
ok, good to know